docs/manuals/core/concepts/sequence.md

```diff
@@ -0,0 +1,32 @@
+A `Sequence^` is designed to model an algorithm. It represents a direct acyclic graph of input, intermediate, and output
+data nodes linked together by tasks. A _sequence_ is a set of tasks designed to perform a set of functions.
```

Comment:
Please use '*' instead of '_' for homogeneity and better support of Python symbols.